markdownlint-cli2.js

```diff
@@ -163,6 +163,23 @@ const readOptionsOrConfig = async (configPath, fs, noRequire) => {
       "(e.g., \".markdownlint.json\" or \"example.markdownlint-cli2.jsonc\")."
     );
   }
+
+  config = (options && options.config) || config;
+
+  if (config && config.extends) {
+    const jsoncParse = await getJsoncParse();
```

Comment:
Is it possible to reuse (instead of duplicate) the existing code for extends?

Reply:
Created a new `getExtendedConfig` function.